src/main/common/filter.c

```diff
@@ -79,22 +72,22 @@ void biquadFilterInitLPF(biquadFilter_t *filter, float filterFreq, uint32_t refr
 {
     biquadFilterInit(filter, filterFreq, refreshRate, BIQUAD_Q, FILTER_LPF);
 }
+
 void biquadFilterInit(biquadFilter_t *filter, float filterFreq, uint32_t refreshRate, float Q, biquadFilterType_e filterType)
 {
     // setup variables
-    const float sampleRate = 1 / ((float)refreshRate * 0.000001f);
-    const float omega = 2 * M_PI_FLOAT * filterFreq / sampleRate;
+    const float omega = 2.0f * M_PI_FLOAT * filterFreq * refreshRate * 0.000001f;
     const float sn = sinf(omega);
     const float cs = cosf(omega);
-    const float alpha = sn / (2 * Q);
+    const float alpha = sn / (2.0f * Q);
 
     float b0 = 0, b1 = 0, b2 = 0, a0 = 0, a1 = 0, a2 = 0;
 
     switch (filterType) {
     case FILTER_LPF:
-        b0 = (1 - cs) / 2;
+        b0 = (1 - cs) * 0.5f;
         b1 = 1 - cs;
-        b2 = (1 - cs) / 2;
+        b2 = (1 - cs) * 0.5f;
```

Comment:
This is a nice find, since multiplication is way faster then division, but the Compiler allready does such stuff for us :)